src/home_robot/home_robot/motion/stretch.py

```diff
@@ -184,26 +189,48 @@ class HelloStretchKinematics(Robot):
         "joint_wrist_roll",
     ]
 
-    def _create_ik_solvers(self, ik_type: str = "pybullet", visualize: bool = False):
+    def _create_ik_solvers(
+        self, ik_type: str = "pybullet", visualize: bool = False
+    ):  # TODO: spowers unused visualize flag?
         """Create ik solvers using pybullet or something else."""
         # You can set one of the visualize flags to true to debug IK issues
         # This is not exposed manually - only one though or it will fail
-        if ik_type == "pybullet":
```

Comment:
self.ik_solver is no longer initialized in main? I think there are still usages of ik_solver though?